### ISSUE TITLE
We misrepresent COADREAD and STES data

### ISSUE THREAD

**Issue by enemeth19** [web4bio/webgen#316]
Tumor type "COADREAD" = tumor type "COAD" + tumor type "READ"
Tumor type "STES" = tumor type "STAD" + tumor type "ESCA"

**Comment by enemeth19** [comment]
It seems that "COADREAD" and "STES" were defined by FireBrowse, as they are not part of the [list of original TCGA study abbreviations](https://gdc.cancer.gov/resources-tcga-users/tcga-code-tables/tcga-study-abbreviations). I didn't realize this until now, and I think that this may also be confusing to users. My thoughts are to remove COADREAD and STES from our list of tumor type options in order to avoid others from having this confusion in the future, and to represent tumor types in their original form, as defined by NCI and GDC.

**Comment by 19aanand** [comment]
Thank you for looking into this Erika! One comment and two questions from me.

My comment is more so to check if I'm understanding this conceptually. The issue is occurring because TCGA does not contain the previously-mentioned tumor type abbreviations (which do exist in Firebrowse). Does this mean that Firebrowse is "making up" tumor type options, or is there a bit more to it?

Also, I am leaning towards preserving the specified tumor type options, but one factor that would make me lean towards removing those tumor types is whether we anticipate any other tumor type options causing the same issue? If so, that may our job more tedious than it needs to be.

**Comment by enemeth19** [comment]
> The issue is occurring because TCGA does not contain the previously-mentioned tumor type abbreviations (which do exist in Firebrowse).

Yes

> Does this mean that Firebrowse is "making up" tumor type options, or is there a bit more to it?

I would use the word "defining" rather than "making up" - essentially, Firebrowse has aggregated data for patients with certain tumor types into new terms

- e.g., Patients with any available cancer of the kidney ("KICH", "KIRCH", "KIRP") are contained within "KIPAN"

> whether we anticipate any other tumor type options causing the same issue?

I'm pretty sure that these are the only tumor types that would be causing this issue, since they are the only tumor types that contain aggregated data

**Comment by 19aanand** [comment]
Thanks for clarifying this Erika! In that case, I think this is an issue definitely worth pursuing since it irons out the prior-mentioned edge cases.

**Comment by kaczmarj** [comment]
>It seems that "COADREAD", "GBMLGG", "KIPAN", and "STES" were defined by FireBrowse, as they are not part of the list of original TCGA study abbreviations.

this is true. tcga does have a publication on colorectal cancer that combines COAD and READ (https://gdc.cancer.gov/about-data/publications/coadread_2012). so it seems that tcga also recognizes the combination of the two.

LGG is a low grade glioma, and GBM is high grade glioma. so together, they represent a spectrum of gliomas. gliomas are a type of brain tumor.

KIPAN is a pan-kidney cohort of cancers (all are related to the kidney. see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6075733/ for the relevant publication
